website/pages/docs/nia/cli.mdx

```diff
@@ -6,6 +6,52 @@ description: >-
   How to use the Consul-Terraform-Sync CLI
 ---
 
-# Consul-Terraform-Sync CLI
+# Consul-Terraform-Sync Command (CLI)
 
-Consul-Terraform-Sync is controlled via as easy to use command-line interface (CLI). Consul-Terraform-Sync is only a single command-line application: consul-terraform-sync.
+Consul-Terraform-Sync is controlled via an easy to use command-line interface (CLI). Consul-Terraform-Sync is only a single command-line application: `consul-terraform-sync`.
+
+Since there is no default configuration to run Consul-Terraform-Sync in a meaningful way, setting a configuration flag `-config-file` or `-config-dir` is required. For example:
+
+```shell-session
+$ consul-terraform-sync -config-file=config.hcl
+```
+
+To view a list of available flags, use the `-help` or `-h` flag. Note that Consul-Terraform-Sync is abbreviated as 'Sync'.
+
+```shell-session
```

Comment:
I'm not sure we should include the cli's -h/-help output in the docs. These can change fairly frequently and are hard to keep in sync. Maybe just leave it out except for a reference to check it. Otherwise having the cli docs only call out the notable/important flags (probably everything but version currently).

Reply:
That makes sense. I was trying to do something like [Consul's cli docs](https://www.consul.io/commands/index#consul-commands-cli) but they were printing out subcommands rather than flags, which won't change as frequently. I'll took out the shell snippet and its deployed on netlify: https://deploy-preview-8840--consul-docs-preview.netlify.app/docs/nia/cli

Reply:
That looks perfect. Thanks.